casadi/core/runtime/casadi_cvx.hpp

```diff
@@ -158,7 +158,7 @@ void casadi_cvx_givens(T1 a, T1 b, T1* c, T1* s) {
 // Golub & Van Loan Alg. 8.3.2
 template<typename T1>
 void casadi_cvx_implicit_qr(casadi_int n, T1* t_diag, T1* t_off, T1* cs) {
-  T1 d, mu, to2, x, z, c, s, t1, t2, d0, d1, o0, o1, sd;
+  T1 d, mu, to2, x, z, c, s, t1, t2, d0, d1, o0, sd;
   casadi_int i;
```

Comment:
@victorfors A general comment: this code isn't readable. There should be comments on each line explaining every step of the algorithm

Reply:
Ok, I will add some comments in this function where it makes sense.

I've been going through some of the code in this file, would you want a pull request with added comments if I find some other places where I think they would help readability?

Reply:
My personal opinion is that all code should be properly commented or be removed from the repo. I'm not sure what the story is for this particular code however. I think pull requests for adding comments are always welcome.

Reply:
Added comments. Sorry for the merge noise in the commit history but I assume you can squash that if you merge.